src/connectors/news/newsApiConnector.test.ts

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios';
+import NewsApiConnector from './newsApiConnector';
+
+jest.mock('axios');
+const resultMock = {
+
+
+  status: 'ok',
+  totalResults: 1,
+  articles: [
+    {
+      source: {
+        id: null,
+        name: 'Chip.de',
+      },
+      author: 'Konstantinos Mitsis',
+      title: 'Aldi, Lidl, Rewe & Co. verschärfen Regeln: Filialen verändern sich wegen Corona massiv - CHIP Online Deutschland',
+      description: 'Die Verunsicherung durch das Coronavirus verändert auch die Art, wie wir einkaufen. Viele Supermärkte und Discounter wie Aldi, Lidl oder Rewe ändern ihr Sortiment und ihre Abläufe. CHIP fasst die wichtigsten Veränderungen zusammen. Worauf Sie beim Einkaufen i…',
+      url: 'https://www.chip.de/news/Aldi-Lidl-Rewe-Co.-verschaerfen-Regeln-Filialen-veraendern-sich-wegen-Corona-massiv_182554531.html',
+      urlToImage: 'https://www.chip.de/ii/1/2/6/2/3/4/3/0/6/nettoabstand3-f0a024d448c3a2ff.jpg',
+      publishedAt: '2020-04-02T17:48:07Z',
+      content: 'Digital und kontaktlos bezahlenIn der Coronakrise bekommen Techniken Aufwind, die es eigentlich schon lange gibt. So nutzen die Deutschen nun verstärkt kontaktlose Bezahlmöglichkeiten mit Karte oder Smartphone. Unser Guide beantwortet auf 29 Seiten alle wicht… [+74 chars]',
+    },
+
+  ],
```

Comment:
Hier kann man vielleicht etwas kürzere Mockdaten (foo, bar etc.) als Werte nehmen, damit der Test kompakter wird. Oder das Objekt selbst mit nur einer Property machen, weil das ja eh ohne Verarbeitung zurückgegeben wird.

Reply:
Ich dachte mir das indirekt als Doku und im normalfall kannst du das in deiner IDE einklappen.